tests/test_modeling_bart.py

```diff
@@ -392,16 +392,22 @@ def _get_embs(m):
         self.assertTrue(torch.eq(input_new, output_new).all())
 
 
-def _assert_tensors_equal(a, b, atol=1e-12, prefix=""):
-    """If tensors not close, or a and b arent both tensors, raise a nice Assertion error."""
+def assert_tensors_close(a, b, atol=1e-12, prefix=""):
+    """If tensors not close, or a and b aren't both tensors, raise a nice Assertion error."""
+
     if a is None and b is None:
         return True
+    assert a.shape == b.shape
```

Comment:
why use `assert` here? Using `self.assertIsEqual` as is done in other test files is better IMO.

Reply:
Or why is it better to use `assert` here?

Reply:
Not sure but I think the goal of this function is mostly for customizing the error message? @sshleifer

Reply:
and it's not in any test class so does not have `assertIsEqual`